Tests/LibGfx/BenchmarkJPEGLoader.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023, Lucas Chollet <lucas.chollet@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/File.h>
+#include <LibGfx/ImageFormats/JPEGLoader.h>
+#include <LibTest/TestCase.h>
+
+#ifdef AK_OS_SERENITY
+#    define TEST_INPUT(x) ("/usr/Tests/LibGfx/test-inputs/" x)
+#else
+#    define TEST_INPUT(x) ("test-inputs/" x)
+#endif
+
+// clang-format off
```

Comment:
why?

Reply:
https://discord.com/channels/830522505605283862/830739873119207426/1091599326746267658

Reply:
I added a comment and a link to the clang-format patch